hack/helm/install-unittest-plugin.sh

```diff
@@ -26,7 +30,7 @@ fi
 
```

Comment:
Maybe we should add a comment here that we use curl because of the api.github request limit to keep it for future knowledge :)